incubator/airflow/templates/deployments-scheduler.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: extensions/v1beta1
```

Comment:
You should use apps/v1, extensions/v1beta1 is deprecated.

Reply:
v1beta1 is required on our kubernetes :(

Reply:
done